src/Indigo/Contracts/HomebaseLite/Impl/Metadata.hs

```diff
@@ -0,0 +1,87 @@
+-- SPDX-FileCopyrightText: 2022 Tezos Commons
+-- SPDX-License-Identifier: LicenseRef-MIT-TC
+
+{-# LANGUAGE OverloadedLists #-}
+
+module Indigo.Contracts.HomebaseLite.Impl.Metadata
+  ( metadataBigMap
+  , versionString
+  , gitRev
+  ) where
+
+import Indigo hiding (description, name, (<>))
+
+import Data.Aeson (encode)
+import qualified Data.ByteString.Lazy as BS
+import Data.Text.Internal.Builder (toLazyText)
+import Language.Haskell.TH.Lib
+import Language.Haskell.TH.Syntax
+
+import qualified Lorentz.Contracts.Spec.TZIP16Interface as TZ16
+import Morley.Metadata (compileViewCodeTH, mkSimpleMichelsonStorageView)
+import Morley.Micheline (toExpression)
+import Morley.Michelson.Text (unsafeMkMText)
+import Morley.Util.Markdown (HeaderLevel(..))
+
+import Indigo.Contracts.HomebaseLite.Impl.Admin ()
+import Indigo.Contracts.HomebaseLite.Impl.Metadata.Views
+import Indigo.Contracts.HomebaseLite.Impl.Utils
+import Indigo.Contracts.HomebaseLite.Impl.Vote ()
+import Indigo.Contracts.HomebaseLite.Types
+
+metadataBigMap :: TZ16.MetadataMap BigMap
+metadataBigMap = TZ16.metadataURI (TZ16.tezosStorageUri TZ16.selfHost metadataKey)
+  <> [(metadataKey, BS.toStrict $ encode metadataJSON)]
+  where metadataKey = [mt|metadata|]
+
+metadataJSON :: TZ16.Metadata (ToT Storage)
+metadataJSON = mconcat
```

Comment:
I believe that this could also have the `authors` (`"Serokell"` and `"Tezos Commons"`) and `interfaces` (`TZIP-16` itself) specified.